Cannot resolvev

`Module not found: Can't resolve 'date-fns/esm/locale' in ...'`

I'm following the i18n example in the docs

```
import { es, ru } from 'date-fns/esm/locale'
```

In a Create React App generated project (ie webpack)

The esm folder does not exist in node_modules and I can only find it as a `module` value in package.json

How should we import locales?

--- a/docs/i18n.md
+++ b/docs/i18n.md
@@ -43,9 +43,9 @@ instead of original functions:
 // app/_lib/format.js
 
 import {format} from 'date-fns/esm'
-import {en, eo, ru} from 'date-fns/esm/locale'
+import {enGB, eo, ru} from 'date-fns/esm/locale'
 
-const locales = {en, eo, ru}
+const locales = {enGB, eo, ru}
 
 export default function (date, formatStr) {
   return format(date, formatStr, {

